vpm-flexible-gallery.php

```diff
@@ -97,13 +97,18 @@ function vpm_custom_gallery_output( $output, $attr, $instance = null ) {
 		$img = wp_get_attachment_image_src( $id, $size );
 		$src = apply_filters( 'vpm_gallery_image_src', $img_full[0], $id, $size );
 
+		// Fetch image caption
+		$caption = $attachment->post_excerpt;
+
 		// Output of image item; default WP structure (dl.gallery-item > dt.gallery-icon > a > img)
-		$gallery_item_html = apply_filters( 'vpm_gallery_item_markup', '<dl class="gallery-item"><dt class="gallery-icon">%1$s</dt></dl>' );
```

Comment:
Can we have these statements aligned?

https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#indentation